client/stylesheets/_modal.scss

```diff
@@ -230,12 +230,12 @@
 }
 
 /* hide input */
-.modOfferRequestPaymentWrap ul li input.offerRequestPaymentRadio:empty {
+.radioWrap ul li input.radioWrapElement:empty {
 	margin-left: -2000px;
 }
 
 /* style label */
-.modOfferRequestPaymentWrap ul li input.offerRequestPaymentRadio:empty ~ label {
+.radioWrap ul li input.radioWrapElement:empty ~ label {
```

Comment:
Merge rule `.radioWrap ul li input.radioWrapElement:empty ~ label` with rule on line 215<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `radioWrap` should be written in lowercase with hyphens<br>Selector `radioWrapElement` should be written in lowercase with hyphens